data/mods/gen8/random-teams.ts

```diff
@@ -304,9 +304,12 @@ export class RandomGen8Teams {
 
 	/**
 	 * Check if user has directly tried to ban/unban/restrict things in a custom battle.
-	 * Doesn't count bans nested inside other formats/rules.
+	 * Doesn't count bans nested inside other formats/rules except Force Select.
 	 */
 	private hasDirectCustomBanlistChanges() {
+		// Force Select is functionally a ban on not including a specific Pokemon.
```

Comment:
This premise is actually wrong. That should only happen in Species Clause formats. Which, granted, Hackmons Cup effectively is, but it's still the case that this is the wrong level to enforce Species Clause.

Reply:
Currently Force Select seems to have no relationship to Species Clause. For example, when creating battles like this...

```
/tour new [Gen 9] Anything Goes, Elimination
/tour rules Force Select=Pikachu
/tour autostart 5
/tour autodq 2
```

...the tour and battles can start. Each player has to include at least one Pikachu on their team, but since there is no Species Clause, they may have more than one each.

Is this behavior a bug? If needed, Force Select could require Species Clause in its onValidateRule to prevent this. But if the current behavior doesn't cause any other bugs, it feels like this would just be cutting out functionality that somebody might want to use.